### ISSUE TITLE
Videos with same title not downloaded under subscriptions.

### ISSUE THREAD

**Issue by heyyceejayy** [Tzahi12345/YoutubeDL-Material#241]
Thanks for your great software, its very useful in archiving youtube videos I care about.

I'm reporting a bug in which your code doesnt download a video if a file exists under the same name. I follow a YouTuber who, for some odd reason decided to reuse a title. I noticed that youtubedl-material refuses to download the new video due to the fact that an existing video under the same name exists in the folder.

So basically the issue is, 2 videos under the same title, different video ids, the older one is retained and the newer one is not downloaded. Hope this makes sense.

**Comment by Tzahi12345** [comment]
This should be easy to solve -- basically the issue is that video files are stored by the video title, and so having the same name would cause issues (though I'd expect the new one to work, and the old one to fail). Either way, you can fix this by going to the subscription and clicking the edit button, and set the _custom file output_ to something like this: `[%(id)s] - %(title)s.%(ext)s`

That will prepend the title with the video ID from youtube, but there's a lot more options besides the video ID ([see here](https://github.com/blackjack4494/yt-dlc#output-template)).  Should prevent any conflicts from occurring in the future. Hope this helps!

**Comment by heyyceejayy** [comment]
Thanks! That did the trick!

However I did find another bug, which I believe I've been running into when a subscription contains a huge number of videos. It doesn't edit the subscription, and I'm seeing a "Payload too large" on the logs. I'll try to replicate, as the database got borked over the weekend somehow and I haven't rebuilt my subscriptions yet.
